### DISCUSSION TITLE
Track card is still rather convoluted

### DISCUSSION THREAD

**Discussion by sonora** [osmandapp/OsmAnd#11960] [General]
I think our track analysis card (the one that opens when you tap a gpx track) still needs an overhaul, it is not very clearly designed:

- There is e.g. a button 'Overview', but on it we mix some overview track data with visibility and appearance confuguration, plus an Edit button. While tapping on some key figures there also leads to some analysis graphs.

- You can also go to the 'Track'  button, and there again you have access to much the same analysis graphs. And it also contains an 'Overview' button, totally unrelated to the other one.

- And then you can get much the same graphs by going to the 'Options' button, then selecting 'Analyze on map'.

- And via that Options button you can also configure the track appearance again, or find another Edit button.

- 'Analyze on map' and 'Options' are also availavle via the 'Track' button. But there 'Options' is just a rudimentary menu, different from the other 'Oprions' button.

All in all I think the entire functionality has way too many tabs and buttons offering things in a duplicate fashion. There is no logical structure, lots of overlap, duplicate wording and duplicate items. We need a redesign with clear distinction between
- Key figures
- Analysis
- Appearance
- Edit

functionality tabs, with no overlap or duplication.

**Comment by dmpr0** [comment]
Options, consist all actions, with text labels.
Track tab: options it options only for one segment/tracks (and this screen is still not if final phrase).
Overview tab consists overview information for all track.

**Comment by vshcherb** [comment]
I think it was discussed many time and it still inline with main strategy. 
- For each option / action should be a clear way to access it i.e. it should **Action with text label** - UI Discoveribility.
- There is no requirement that each action should be accessed only in 1 way! - UX for advanced usage.
- If it's possible to introduce shortcut to certain function / action i.e. clicking on descent label / graph and it looks logical, shortcut should be introduced to improve day-by-day usage. - UX for advanced usage
- There is no need to hide all possible data i.e. show only distance for track without descent / ascent cause useful information might not be accessed via complex settings.

So as the summary it leads to what you describe and it's not considered to be a problem:
- Some actions could be accessed via different menus
- Some actions could be accessed in non-transparent way (but at least 1 way should be transparent)
- Even default UI could have many textual information about place / track / etc

Logical structure:
- Overview - Description / Stats / General information and Quick Access buttons (Show/hide, Appearance)
- Track - Detailed information about track / line  with graph
- Points - All about track points
- Options - All possible actions
